queue_job/migrations/13.0.3.7.0/pre-migration.py

```diff
@@ -0,0 +1,35 @@
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html)
+
+import logging
+
+from odoo.tools.sql import column_exists
+
+_logger = logging.getLogger(__name__)
+
+
+def migrate(cr, version):
```

Comment:
bump version manual and merge these scripts to run in one version for v14

Reply:
ok, thank you. I'll update the script and have a fix for the failed travis

Reply:
@simahawk the PR has been updated